architecture/control-plane.adoc

```diff
@@ -12,4 +12,8 @@ include::modules/operators-overview.adoc[leveloffset=+2]
 
 include::modules/update-service-overview.adoc[leveloffset=+3]
 
-include::modules/understanding-machine-config-operator.adoc[leveloffset=+3]
\ No newline at end of file
+include::modules/understanding-machine-config-operator.adoc[leveloffset=+3]
+
+.Additional information
+
+For information on pausing control plane rebooting after the Machine Congig Operator makes changes to the machine config, see xref:../support/troubleshooting/troubleshooting-operator-issues.adoc#troubleshooting-disabling-autoreboot-mco_troubleshooting-operator-issues[Disabling Machine Config Operator from automatically rebooting].
```

Comment:
```suggestion
For information on temporarily preventing the control plane machines from rebooting after the Machine Congig Operator makes changes to the machine config, see xref:../support/troubleshooting/troubleshooting-operator-issues.adoc#troubleshooting-disabling-autoreboot-mco_troubleshooting-operator-issues[Disabling Machine Config Operator from automatically rebooting].
```
I am not sure that the current phrase will translate well, so maybe something like this would be better?

Reply:
@kalexand-rh I like that, thanks. However, I might remove _temporary_, as the procedure is permanent until you actively revert the parameter.